src/Fixer/LanguageConstruct/FunctionToConstantFixer.php

```diff
@@ -267,12 +273,15 @@ private function fixGetClassCall(
         return null;
     }
 
+    /**
+     * @return array{int, int, list<Token>}
```

Comment:
Repeated 3 times, maybe a candidate for `@phpstan-type`?

Reply:
very small type, i rather focused to extract complex object with 10 keys :D